artemis/reporting/modules/domain_expiration_scanner/translations/pl_PL/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,7 @@
+#: artemis/reporting/modules/domain_expiration_scanner/template_close_domain_expiration_scanner.jinja2:2
+msgid "The following domains will soon expire: "
+msgstr "Poniższe domeny wkrótce wygasną: "
+
+#: artemis/reporting/modules/domain_expiration_scanner/template_close_domain_expiration_scanner.jinja2:7
+msgid " will expire on "
```

Comment:
you translate a version with spase to a version without spaces, that will cause typos